### ISSUE TITLE
Cannot set flarum/core version

### ISSUE THREAD

**Issue by farvardin** [YunoHost-Apps/flarum_ynh#122]
**Describe the bug**
*shell command return code: 1: I can't install the devel version of Flarum*

**Versions**
- YunoHost version:yunohost    3.6.4.6 (stable)
- Package version/branch: *latest* I've installed flarum from the latest github version: https://github.com/YunoHost-Apps/flarum_ynh

**To Reproduce**
*Steps to reproduce the behavior.*
- *I don't have access to the cli, I've installed flarum from the webadmin, and got the error described there:* https://paste.yunohost.org/raw/wasaboxuzu

**Logs**
*see https://paste.yunohost.org/raw/wasaboxuzu*

**Comment by tituspijean** [comment]
Issue lies here:

```
flarum php7.3 /var/www/flarum/composer.phar require flarum/core:0.1.0-beta.9 -n --ansi -d /tmp/flarum -d /var/www/flarum --quiet --no-interaction 2019-09-21 20:22:05,461: WARNING - [37;41mYour requirements could not be resolved to an installable set of packages.
```

It seems we cannot set the core version like that. I'm investigating whether or not this is actually needed.

**Comment by tituspijean** [comment]
@farvardin it should be fixed now, don't hesitate to reopen if the issue persists or open another issue if you face a different error.

**Comment by farvardin** [comment]
Merci beaucoup !
Thank you, it's working now.
For the record, I've tried to install it again with all the options (bazaar and French language), it failed with those errors: https://paste.yunohost.org/raw/penuzuheda
Then I installed it again without bazaar (in French), it worked. I uninstalled, and installed with bazaar and French again, and it worked again. I don't understand why it didn't work in the first place.
